lime-cli/src/main/scala/org/bdgenomics/lime/cli/Cluster.scala

```diff
@@ -27,7 +27,7 @@ object Cluster extends BDGCommandCompanion {
     override var asSingleFile: Boolean = false
     override var deferMerging: Boolean = false
     override var outputPath: String = ""
-    override var disableFastConcat: Boolean = false
+    //override var disableFastConcat: Boolean = false
```

Comment:
oops this was to allow it to build when i was testing sorry abt that will revert